scylla-cql/src/types/deserialize/mod.rs

```diff
@@ -46,30 +46,31 @@
 //!
 //! ```rust
 //! # use scylla_cql::frame::response::result::ColumnType;
-//! # use scylla_cql::frame::frame_errors::ParseError;
 //! # use scylla_cql::types::deserialize::{DeserializationError, FrameSlice, TypeCheckError};
 //! # use scylla_cql::types::deserialize::value::DeserializeValue;
+//! # use thiserror::Error;
```

Comment:
Let's not hide this (don't use `#`), because otherwise readers can be confused about the `Error` trait - where it comes from.